msw/build-nsi.sh

```diff
@@ -104,6 +104,18 @@ then
   cleanup 1
 fi
 
+# autodetect wishname if none is given
+if [ -z "${WISHNAME}" ]; then
+	WISHNAME=$(find "${PDWINDIR}/bin" -iname "wish*.exe" -quit)
```

Comment:
done via b8cae3a

Reply:
ouch. i think the correct fix would be
```
WISHNAME=$(find "${PDWINDIR}/bin" -iname "wish*.exe" -print -quit)
```

the idea is that we are only interested in a single `wish*.exe` (just in case there are multiple files matching that globbing pattern)

Reply:
fine. it works here on Windows and Linux.